docs/homescreen.md

```diff
@@ -49,4 +49,33 @@ The alarm widget is displayed in top left corner after the timer widget. It will
 
 The media player widget is displayed in the bottom of the home screen, It replaces the examples widget when a media player is active. The media player widget will show you the currently playing media and provide you with a quick way to pause, resume or skip the current media. You can also quickly access the media player by tapping the quick display media player button on the right side of the media player widget.
 
-![](https://github.com/OpenVoiceOS/ovos_assets/raw/master/Images/homescreen-mediawidget.gif)
\ No newline at end of file
+![](https://github.com/OpenVoiceOS/ovos_assets/raw/master/Images/homescreen-mediawidget.gif)
+
+## Configuration
+
+### Settings
+
+The homescreen has several customizations available.  This is sample `settings.conf` file with all of the options explained
+
+```
+{
+    "__mycroft_skill_firstrun": false,  # This is set on first load of the skill
```

Comment:
With settings being `json`, I'm not sure if the comments are valid here?

Reply:
Comments are there because it is depreciated. I will move them out of the code

Reply:
current format look good to me